c/tskit/core.c

```diff
@@ -1278,96 +1278,111 @@ tsk_bit_array_init(tsk_bit_array_t *self, tsk_size_t num_bits, tsk_size_t length
     return ret;
 }
 
-void
-tsk_bit_array_get_row(const tsk_bit_array_t *self, tsk_size_t row, tsk_bit_array_t *out)
-{
-    out->size = self->size;
-    out->data = self->data + (row * self->size);
-}
+#define BITSET_DATA_ROW(bs, row) (bs)->data + (row) * (bs)->row_len
```

Comment:
Again, safer to wrap full expression of the macro in parantheses as unexpected things can happen if used with different precedence operators.